openmdao/docs/features/building_blocks/components/exec_comp.rst

```diff
@@ -5,63 +5,60 @@
-=========================  ====================================
-abs(x)                     Absolute value of x
-acos(x)                    Inverse cosine of x
-acosh(x)                   Inverse hyperbolic cosine of x
-arange(start, stop, step)  Array creation
-arccos(x)                  Inverse cosine of x
-arccosh(x)                 Inverse hyperbolic cosine of x
-arcsin(x)                  Inverse sine of x
-arcsinh(x)                 Inverse hyperbolic sine of x
-arctan(x)                  Inverse tangent of x
-asin(x)                    Inverse sine of x
-asinh(x)                   Inverse hyperbolic sine of x
-atan(x)                    Inverse tangent of x
-cos(x)                     Cosine of x
-cosh(x)                    Hyperbolic cosine of x
-dot(x, y)                  Dot-product of x and y
-e                          Euler's number
-erf(x)                     Error function
-erfc(x)                    Complementary error function
-exp(x)                     Exponential function
-expm1(x)                   exp(x) - 1
-factorial(x)               Factorial of all numbers in x
-fmax(x, y)                 Element-wise maximum of x and y
-fmin(x, y)                 Element-wise minimum of x and y
-inner(x, y)                Inner product of arrays x and y
-isinf(x)                   Element-wise detection of np.inf
-isnan(x)                   Element-wise detection of np.nan
-kron(x, y)                 Kronecker product of arrays x and y
-linspace(x, y, N)          Numpy linear spaced array creation
-log(x)                     Natural logarithm of x
-log10(x)                   Base-10 logarithm of x
-log1p(x)                   log(1+x)
-matmul(x, y)               Matrix multiplication of x and y
-maximum(x, y)              Element-wise maximum of x and y
-minimum(x, y)              Element-wise minimum of x and y
-ones(N)                    Create an array of ones
-outer(x, y)                Outer product of x and y
-pi                         Pi
-power(x, y)                Element-wise x**y
-prod(x)                    The product of all elements in x
-sin(x)                     Sine of x
-sinh(x)                    Hyperbolic sine of x
-sum(x)                     The sum of all elements in x
-tan(x)                     Tangent of x
-tanh(x)                    Hyperbolic tangent of x
-tensordot(x, y)            Tensor dot product of x and y
-zeros(N)                   Create an array of zeros
-=========================  ====================================
+it with an equation.
+
+ExecComp Constructor
+--------------------
+
+The call signature for the `ExecComp` constructor is:
+
+.. automethod:: openmdao.components.exec_comp.ExecComp.__init__
+    :noindex:
+
+ExecComp Variable Metadata
+--------------------------
+
+The values of the `kwargs` can be `dicts` which define the initial value for the variables along with
+other metadata. For example,
+
+.. code-block:: python
+
+    ExecComp('y=x', x={'units': 'ft'}, y={'units': 'm'})
```

Comment:
I am not sure I like this little example, because it shows someone intentionally making an incorrect unit conversion. A better example might be `ExecComp('xdot=x/t', x={'units': 'ft'}, t={'units': 's'}, xdot={'units': 'ft/s')`

Reply:
Good suggestion. Done